pages/about/components/Gallery.tsx

```diff
@@ -8,12 +8,18 @@ export const Gallery: FC = ({}) => {
 	return (
 		<>
 			<div className={styles.gallery}>
-				<Image className={styles.about1} src={about1} alt="" />
-				<Image className={styles.about2} src={about2} alt="" />
-				<Image className={styles.about3} src={about3} alt="" />
-				<Image className={styles.about4} src={about4} alt="" />
+				<Image className={styles.about1} src={about1} alt="about1" />
+				<Image className={styles.about2} src={about2} alt="about2" />
+				<Image className={styles.about3} src={about3} alt="about3" />
+				<Image className={styles.about4} src={about4} alt="about4" />
```

Comment:
keep these `alt` text empty for now. It will be fixed later when more suitable image will be put